core/storage/feedback/gae_models.py

```diff
@@ -287,13 +404,29 @@ def create(cls, exploration_id, thread_id, author_id, exploration_version,
 
     @classmethod
     def _get_instance_id(cls, exploration_id, thread_id):
+        """Returns ID of instance the exploration and thread belongs to.
```

Comment:
Returns the full thread ID corresponding to the given exploration ID and thread ID.

(There is no SuggestionModel instance here.)